### ISSUE TITLE
Chapters not in order

### ISSUE THREAD

**Issue by Jkacasio** [Goodwy/PlayBook#32]
### Overview

It is not playing in order. I checked the file naming and I find it correct with 01 as start and so on. And it plays starting with Chapter 10 as 10.

### How to reproduce

When playing

### Version

2.0.0

### Android Version

29

### Device Model

BLA-L29

**Comment by Jkacasio** [comment]
Is there a way to resolve this? I have tried it on another app and it worked fine and are in order. Also did not face issues before in this app before the big update. 
Thank you!


Sent from Yahoo Mail for iPhone


On Tuesday, March 7, 2023, 4:37 PM, Goodwy ***@***.***> wrote:




Inside the files there is a tag responsible for the order of the chapters. It may not have been filled in correctly. File names are not used to determine the order of chapters.

**Comment by Goodwy** [comment]
Can you send the problem file?

**Comment by Jkacasio** [comment]
Can I send a link instead?


Sent from Yahoo Mail for iPhone


On Tuesday, March 7, 2023, 4:56 PM, Goodwy ***@***.***> wrote:




Can you send the problem file?

**Comment by Goodwy** [comment]
Send link to goodwy.dew@gmail.com mail

**Comment by Jkacasio** [comment]
Hi,
Actually it’s in order now. Weird because I swear it was not in order when I played it last night and even the last book I played and completed had 2 chapters placed in a wrong order. 

What I did now was restarted the folders I was synching and it’s now okay. Maybe it just needs a resyncing? 
Also, I noticed that in new update, the app is not automatically refreshing new books added. Like I am adding new sub folder to my folder and it’s not showing. I have to add folder again which I didn’t do in the older version. Maybe a refresh button can be added? Or better it automatically refresh once new books added like before. 

That’s all, I just wanna say I love your app and I hope it will be better again. Thank you! 
Sent from Yahoo Mail for iPhone


On Tuesday, March 7, 2023, 5:18 PM, Goodwy ***@***.***> wrote:




Send link to ***@***.*** mail
